testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/CliConfigUtils.java

```diff
@@ -0,0 +1,101 @@
+package org.jboss.as.test.integration.management.cli;
+
+import org.jboss.as.cli.impl.Namespace;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+
+import static org.junit.Assert.fail;
+
+public class CliConfigUtils {
```

Comment:
I am not sure we really need a new class there. Do you plan to refactor other tests to benefit from this refactoring?

Reply:
I think only CliConfigTestCase.java and LongOutputTestCase.java would benefit from this, since only those 2 would use it. I proposed this change to avoid the duplicate code of the createConfigFile() and writeColorConfig() methods in those 2 tests. Could we leave those methods in the CliConfigTestCase and increase their visibility so they can be reused in the LongOutputTestCase ?